nightly/pytest-sanity.txt

```diff
@@ -143,4 +143,7 @@ pytest sanity/recompress_storage.py
 pytest sanity/recompress_storage.py --features nightly
 
 # This is the test for meta transactions.
-pytest sanity/meta_tx.py --features nightly
\ No newline at end of file
+pytest sanity/meta_tx.py --features nightly
+
+# Tests for split storage and split storage migration
+pytest sanity/split_storage.py
```

Comment:
should we also add it for nightly?

Reply:
What do you mean? If you mean the nightly.txt in the pytest directory, it looks like it already references the sanity tests as well via pytest.txt -> pytest-sanity.txt.

Reply:
I meant adding:

pytest sanity/split_storage.py --featuers nightly 

(this would run it with new features - for example flat storage)